test/scripts/common.inc

```diff
@@ -0,0 +1,63 @@
+#
+# Include file with common parts shared for local and ephemeral
+#
+
+# Troubleshooting:
+# - Run with DEBUG=1 to see some traces from curl.sh wrapper
+#   $ DEBUG=1 ./test/scripts/local-domain-token.sh
+# - Run with more verbose by:
+#   $ DEBUG=1 bash -xv ./test/scripts/local-domain-token.sh
+
+set -eo pipefail
```

Comment:
Shell settings are typically set in the beginning of a shell script, not in an include source file. It allows a developer to source an include into an interactive session without setting `-e`.